configs/eth_v4.json

```diff
@@ -0,0 +1,17 @@
+{
+    "config": {
+        "network-interface":{
+            "help": "options are ETHERNET,WIFI,MESH_LOWPAN_ND,MESH_THREAD",
+            "value": "ETHERNET"
+        }
+    },
+    "macros": ["MBEDTLS_USER_CONFIG_FILE=\"mbedtls_mbed_client_config.h\""],
+    "target_overrides": {
+        "*": {
+            "target.features_add": ["LWIP", "NANOSTACK", "COMMON_PAL"],
```

Comment:
Unfortunately, it does! If one does NOT include Nanostack if fails:

jankii01@ubuntu:~/mbed/mbed-os-example-client$ ./build_all.sh 
Building project mbed-os-example-client (K64F, GCC_ARM)
Scan: .
Scan: FEATURE_LWIP
Scan: FEATURE_UVISOR
Scan: FEATURE_COMMON_PAL
Scan: FEATURE_BLE
Scan: FEATURE_STORAGE
Scan: FEATURE_THREAD_BORDER_ROUTER
Scan: FEATURE_THREAD_ROUTER
Scan: FEATURE_LOWPAN_BORDER_ROUTER
Scan: FEATURE_LOWPAN_ROUTER
Scan: FEATURE_LOWPAN_HOST
Scan: FEATURE_NANOSTACK_FULL
Scan: FEATURE_NANOSTACK
Scan: FEATURE_THREAD_END_DEVICE
Scan: mbed
Scan: env
Compile [  0.3%]: NanostackRfPhyAtmel.cpp
[ERROR] ./atmel-rf-driver/source/NanostackRfPhyAtmel.cpp:18:44: fatal error: nanostack/platform/arm_hal_phy.h: No such file or directory
compilation terminated.

-> it only works w/o NANOSTACK - IF YOU MOVE OUT THE other extra repos. This is due to the "compile all" nature of the mbed! If you move the extra RF-drivers to TARGET_IGNORE -folder, then it works. But that's another pain.